### ISSUE TITLE
kendo-react-dropdowns doesn't work with react-create-app + typescript.

### ISSUE THREAD

**Issue by shadowmint** [telerik/kendo-react#136]
## I'm submitting a...

* Bug report

## Current behavior

kendo-react-dropdowns does not work with create-react-app when using typescript.

> node_modules/@progress/kendo-react-dropdowns/dist/npm/common/settings.d.ts
>  Type error: Ambient const enums are not allowed when the '--isolatedModules' flag is provided.  TS1209

## Expected behavior

Using `<DropDownList>` should not cause an error.

## Minimal reproduction of the problem with instructions

```
npx create-react-app foo
cd foo
```

Now follow the guide here to enable typescript:

https://facebook.github.io/create-react-app/docs/adding-typescript

Note the comment at the end of the documentation:

> Note: Constant enums and namespaces are not supported.

Now attempt to use kendo-react-dropdowns:

```
npm install @progress/kendo-react-dropdowns
```

...

```
import {DropDownList} from "@progress/kendo-react-dropdowns";
```

Gives:

```
node_modules/@progress/kendo-react-dropdowns/dist/npm/common/settings.d.ts
Type error: Ambient const enums are not allowed when the '--isolatedModules' flag is provided.  TS1209

     98 |  * @hidden
     99 |  */
  > 100 | export declare const enum ActiveDescendant {
        |                           ^
    101 |     'PopupList' = 0,
    102 |     'TagsList' = 1
    103 | }
```

## What is the motivation or use case for changing the behavior?

react-create-app is the standard way of create react applications, and have selected the babel typescript integration as the 'blessed' way of using typescript.

Certainly, the typescript standard supports this feature, but `DropDownList` is a fundamental component; this is a show-stopper for using kendo-react.

## Environment

Package versions:
```
    "@progress/kendo-react-buttons": "^2.4.0",
    "@progress/kendo-react-dateinputs": "^2.4.0",
    "@progress/kendo-react-dropdowns": "^2.4.0",
    "@progress/kendo-react-inputs": "^2.4.0",
    "@progress/kendo-react-intl": "^2.4.0",
    "@progress/kendo-theme-material": "^1.5.1",
    "@progress/kendo-ui": "^2018.3.1129",
    "@types/jest": "^23.3.10",
    "@types/node": "^10.12.10",
    "@types/react": "^16.7.8",
    "@types/react-dom": "^16.0.11",
    "node-sass": "^4.10.0",
    "react": "^16.6.3",
    "react-dom": "^16.6.3",
    "react-scripts": "2.1.1",
    "typescript": "^3.1.6"
```

Browser:
Since this prevents compile, it affects all browsers.

System:
- TypeScript version: 3.1.6
- Node version: v10.13.0
- Platform:  Windows

**Comment by vladimirivanoviliev** [comment]
Thank you for reporting this issue - we will investigate how to improve this behavior in next releases. Until fix is released you can fix the issue as suggestted in the following thread:

[TypeScript/issues/20703](https://github.com/Microsoft/TypeScript/issues/20703#issuecomment-437291032)

**Comment by nstoychev** [comment]
@shadowmint - We have published a fix in our dev channel [2.5.0-dev.201812030901](https://www.npmjs.com/package/@progress/kendo-react-grid/v/2.5.0-dev.201812030901). See the following article about obtaining development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.

**Comment by Xizario** [comment]
`v2.6.0` is published.
